Header header
string anchor_id
geometry_msgs/Pose pose